cluster-provision/manifests/calico.yaml

```diff
@@ -0,0 +1,793 @@
+---
```

Comment:
Please add a note that this file is copied from Calico documentation and that no changes should be done on it (we have the `sed` replacing the default route and that is good, when we want a new version of calico, we just copy a new file from docs, no need to adjust it here.

Reply:
done

Reply:
Can we "curl" it ?

Reply:
ok let's n ot curl it.